source/patterns/@aws-solutions-constructs/aws-wafwebacl-apigateway/README.md

```diff
@@ -0,0 +1,104 @@
+# aws-wafwebacl-apigateway module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_wafwebacl_apigateway`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-wafwebacl-apigateway`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.wafwebaclapigateway`|
+
+## Overview
+This AWS Solutions Construct implements an AWS WAF connected to Amazon API Gateway REST API.
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import * as api from '@aws-cdk/aws-apigateway';
+import * as lambda from "@aws-cdk/aws-lambda";
+import { WafWebACLToApiGatewayProps, WafWebACLToApiGateway } from "@aws-solutions-constructs/aws-wafwebacl-apigateway";
+
+const lambdaProps: lambda.FunctionProps = {
+    code: lambda.Code.fromAsset(`${__dirname}/lambda`),
+    runtime: lambda.Runtime.NODEJS_12_X,
+    handler: 'index.handler'
+};
+
+const lambdafunction = new lambda.Function(this, 'LambdaFunction', lambdaProps);
+const apiGateway = new api.RestApi(this, 'test-api');
+const tasks = apiGateway.root.addResource('tasks');
+
+apiGateway.addMethod('GET', new api.LambdaIntegration(lambdaFunction));
```

Comment:
Why don't we just use our own aws-apigateway-lambda construct here rather than lower level CDK commands?